### PR TITLE
To do feature day-1

### PR DESCRIPTION
**Pull request by gergitokusse**
## 👇 The following file was added to the project 👇 

- ➡️ Add index.html as a template
- ➡️ Add css and javascript
- ➡️ Add new module

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @gergitokusse, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a readme file that is not fully descriptive. Kindly remove the section descriptions and replace them with your own information about the project
image
image
image
- [ ] Also you have the `Live Demo` link section with a link that leads nowhere, kindly remove this section if you choose to not deploy your project, and be sure to remove it from the table of contents as well
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AlphaNtihinduka — COMMENTED** [review]
Hi @gergitokusse 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
# Invalid Code Review Request ❌ 

- [ ] You have submitted a project with a readme file that is not detailed. Kindly edit and customize your readme with the project information for it to be professional. For example the setup and run tests subsections should have information that is related only to the project. And also, the [prerequisites](https://www.google.com/search?q=prerequisites&oq=prerequisites&aqs=chrome..69i57j0i512l9.421j0j7&sourceid=chrome&ie=UTF-8) section should have information instead of a clone link
- [ ] In addition, the PR description is not descriptive to the project requirements.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏.

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by gergitokusse** [comment]
## Hi @AlphaNtihinduka , could you check it. I have modified the readme file.
🙏🙏🙏 Thank you very much.

File: src/index.html [hunk 1]
```diff
@@ -3,9 +3,47 @@
...
+                </div>
+                <div>
+                    <span class="desc">
+                        <input type="checkbox" id="1"/>
+                        <span>My list-1</span>
+                    </span>
+                    <span class="more">
+                        M
+                    </span>
+                </div> -->
```
Comment (afanimartin) [inline src/index.html]: - Please remove this comment. Comments make our code look ugly.

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-  return element;
-}
-document.body.appendChild(component());
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', () => Storage.displaytodo());
+document.addEventListener('DOMContentLoaded', () => Storage.checkboxevent());
+
+const describ = document.querySelector('.enter-todo');
+const form = document.querySelector('.form');
+form.addEventListener('submit', () => Storage.Addtodo(describ.value));
```
Comment (afanimartin) [inline src/index.js]: - Please add a function to populate the interface with sample todo items. First, create an array with sample items.
You could refer to the code below:
```javascript
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```
#### NB: The items should appear dynamically on page reload in the order of their indexes.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @gergitokusse, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- `webpack` is used to serve the files as required
- `HTML` file configured in webpack plugin
- Professional readme file template used

## Required Changes ♻️
_Check the comments under the review._
- Please fix your website. Make sure the todo items render on page load.
Screenshot from 2023-05-11 14-51-19


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (gergitokusse) [reply src/index.html]: Okay, I will remove it, i just use it for checking the layout of JS added elements.

File: src/index.js (hunk 2 above)
Comment (gergitokusse) [reply src/index.js]: Okay thank you I will do it

File: src/index.js [hunk 3]
```diff
@@ -1,12 +1,45 @@
...
+  span3.className = 'more';
+
+  const btn2 = document.createElement('span');
+  btn2.innerHTML = `
+  <i class="bi-three-dots-vertical" />        
+  `;
+  span3.appendChild(btn2);
+  div.appendChild(span3);
+  todo.appendChild(div);
+});
```
Comment (AbelG101) [inline src/index.js]: - [ ] Ouch! this is close but as specified in the project requirements the list should appear in order of the index values for each task. So when you iterate over each task you have to make sure they are sorted by their index values first.
- Project requirement 👇
  image
- I noticed you've placed the same index values for each task please make sure the index value for each task object is different as they'll be used to order the items.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @gergitokusse 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 3 above)
Comment (gergitokusse) [reply src/index.js]: ## Okay, really sorry, I will fix it

**Review by AbelG101 — APPROVED** [review]
# Hi @gergitokusse 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**